Add deb package. Fixes #38

NicolaiSoeborg opened the pull request:
Simple instructions for using PPA to install termshark

NicolaiSoeborg committed:
Add deb package. Fixes #38

pocc commented on README.md:
````diff
@@ -35,6 +35,14 @@ To build/install the port, run:
 
 ```cd /usr/ports/net/termshark/ && make install clean```
 
+# Ubuntu
````
Use ## to match the level-two headers for Arch and FreeBSD.

pocc commented on README.md:
```diff
@@ -35,6 +35,14 @@ To build/install the port, run:
+
+You can use the PPA *nicolais/termshark* to install termshark:
+
+ * `sudo add-apt-repository ppa:nicolais/termshark`
```
Install instructions should be a code block that includes `sudo apt update`. Language hinting is preferred (in this case, sh).

\`\`\`sh
...
\`\`\`

NicolaiSoeborg replied:
I've changed the comment to use `-` instead of a `*` (as with the other lines), but I think the language hinting might only work for "full code blocks" (not single lines in a list?).

pocc commented on README.md:
```diff
@@ -35,6 +35,14 @@ To build/install the port, run:
+
+You can use the PPA *nicolais/termshark* to install termshark:
+
+ * `sudo add-apt-repository ppa:nicolais/termshark`
```
I'm getting this error when I download in my Ubuntu 18.04 VM:

```bash
E: The repository 'http://ppa.launchpad.net/nicolais/termshark/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

(moving to PR comment)

pocc reviewed (changes requested):
Please fix these issues and we can merge!

pocc commented:
I'm getting this error when I download in my Ubuntu 18.04 VM:

```bash
E: The repository 'http://ppa.launchpad.net/nicolais/termshark/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

gcla commented:
Thanks for putting this together :-) Just to confirm I see the same error as @pocc - so then apt search termshark doesn't find it. 

I just discovered it's packaged in debian unstable now too: https://packages.debian.org/sid/main/termshark

and may be coming in Ubuntu 19.10: https://packages.ubuntu.com/source/eoan/termshark

But a PPA makes it easy in the meantime.

NicolaiSoeborg committed:
Fixes for PR #40

NicolaiSoeborg commented:
Awesome if it is going to be included in the official repository soon(-ish?), then this PR might not be needed after all.

This PR was made in a bit of a hurry, but I've fixed the comments regarding README.md and changed the PPA to include the binary for all supported distributions (`Eoan`, `Disco`, `Cosmic`, `Bionic`, `Xenial`, `Trusty`, and `Precise`).

gcla commented:
Merged, thank you!

dawidd6 commented:
Btw @gcla I am the maintainer of this package in Debian, so if there are any problems with it in the future, you can simply tag me :)

gcla commented:
thanks @dawidd6
